### ISSUE TITLE
quality files issues

### ISSUE THREAD

**Issue by RomainJP** [PennBBL/xcpEngine#123]
Hi,
 I constantly have issues with the quality.csv files. At the subjects level, the csv files exist but show repetition of certain measures, sometimes 2, 3 times sometimes 5 times  etc .... at the group level the quality file is always empty.
[accelerator_rduprat_201807271721_ZAPR01_C868_sp120_PreIFGLOG.txt](https://github.com/PennBBL/xcpEngine/files/2274606/accelerator_rduprat_201807271721_ZAPR01_C868_sp120_PreIFGLOG.txt)
[dsn_sp120_173v.txt](https://github.com/PennBBL/xcpEngine/files/2274608/dsn_sp120_173v.txt)
[ZAPR01_C868_sp120_PreIFG_quality.txt](https://github.com/PennBBL/xcpEngine/files/2274612/ZAPR01_C868_sp120_PreIFG_quality.txt)

Any idea why?

**Comment by mattcieslak** [comment]
Hi Romain,

Which columns are repeated in ZAPR01_C868_sp120_PreIFG_quality.txt? I see the header listing 

```
id0,pctSpikesRMS,meanDV,pctSpikesDV,relMeanRMSMotion,nSpikesRMS,nSpikesDV,relMaxRMSMotion,coregCrossCorr,coregJaccard,coregDice,coregCoverage,normJaccard,normDice,normCrossCorr,normCoverage
```

which are all unique

**Comment by RomainJP** [comment]
If you check, there should be more values than headers and I assume it because some columns are duplicated:"0,.61764705882352941176,0,.61764705882352941176,1.216939,.02923976608187134502,.02923976608187134502"

**Comment by RomainJP** [comment]
Should I go for the v0.6.1?

**Comment by mattcieslak** [comment]
You could get the fix by doing 

```console
$ git clone git@github.com:PennBBL/xcpEngine.git
$ cd xcpEngine
$ export XCPEDIR=`pwd`
$ source xcpReset
```

and then re-running your pipeline. Let me know if it works

**Comment by RomainJP** [comment]
I'll try and keep you posted.
Thanks

**Comment by RomainJP** [comment]
Hi Matt, this actually cause the pipeline to crash and I cant make it work again and I need it to process some data this weekend. I forgot to mention we have our own instal of xcp, maybe that's the issue? but the pipeline needs to work this weekend.

**Comment by sattertt** [comment]
Hi Roman, we are happy to help you but sadly need more lead to help fix this.  If you have an urgent deadline in future would try to give us a heads up early on.

Sent from my iPhone

On Aug 10, 2018, at 5:02 PM, Romain <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi Matt, this actually cause the pipeline to crash and I cant make it work again and I need it to process some data this weekend. I forgot to mention we have our own instal of xcp, maybe that's the issue? but the pipeline needs to work this weekend.

**Comment by mattcieslak** [comment]
What does crash mean? Could you post an error message? What is the command you use to run your pipeline? It's possible to revert to an old version by cd-ing into your previous xcp directory, then running 

```console
$ export XCPEDIR=`pwd`
$ source xcpReset
```

I think I have access to your code on the cluster and can fix this manually this time, but we'll need to set up a git-tracked installation for debugging in the future.

**Comment by RomainJP** [comment]
Hi Matt, the pipeline was running fine but after I ran the command it just
goes through all the steps in 1 sec, I managed to fix it though and the
scanner on monday is canceled so no rush.

Thanks for your help, I ll come back to you on Monday.

Romain Duprat, Ph.D.
Postdoctoral Fellow
Center for Neuromodulation in Depression and Stress
University of Pennsylvania, Perelman School of Medicine
Richards Biomedical Building, D303
3700 Hamilton Walk
Philadelphia, PA 19104
Phone: 215-573-8888

**Comment by sattertt** [comment]
Oh good. Glad to hear it.

Also FYI— Matt is working on a containerized version which will obviate a lot of this.  Should be done soon.

Sent from my iPhone

On Aug 10, 2018, at 6:17 PM, Romain <notifications@github.com<mailto:notifications@github.com>> wrote:

Hi Matt, the pipeline was running fine but after I ran the command it just
goes through all the steps in 1 sec, I managed to fix it though and the
scanner on monday is canceled so no rush.

Thanks for your help, I ll come back to you on Monday.

Romain Duprat, Ph.D.
Postdoctoral Fellow
Center for Neuromodulation in Depression and Stress
University of Pennsylvania, Perelman School of Medicine
Richards Biomedical Building, D303
3700 Hamilton Walk
Philadelphia, PA 19104
Phone: 215-573-8888
